crates/primitives/src/transaction/eip4844.rs

```diff
@@ -46,9 +46,8 @@ pub struct TxEip4844 {
     ///
     /// This is also known as `GasTipCap`
     pub max_priority_fee_per_gas: u128,
-    /// The 160-bit address of the message call’s recipient or, for a contract creation
-    /// transaction, ∅, used here to denote the only member of B0 ; formally Tt.
-    pub to: TxKind,
+    /// The 160-bit address of the message call’s recipient.
+    pub to: Address,
```

Comment:
@joshieDo please check if this is sound:

ref https://github.com/paradigmxyz/reth/pull/7276#issuecomment-2032791334